README.rst

```diff
@@ -129,6 +129,12 @@ For plotting
 - `matplotlib <http://matplotlib.org/>`__:
   required for plotting.
 
+Miscellaneous
+~~~~~~~~~~~~~
+
+- `pydantic <https://github.com/samuelcolvin/pydantic>`__:
+  required to use `ConstrainedSession`.
```

Comment:
We probably want to add it to larrayenv, no?

Reply:
Hum... Yes, indeed.
I already added pydantic to the list of packages in the function update_metapackage()

Reply:
Sorry, I missed that 😞